modules/tinymce/src/core/main/ts/api/Env.ts

```diff
@@ -248,6 +248,12 @@ const Env: Env = {
 
   cacheSuffix: null,
   container: null,
+
+  /**
+   * <em>Deprecated in TinyMCE 5.5</em> - if you need the original target of an event, please use `event.composedPath()`
+   *
+   * @deprecated
+   */
   experimentalShadowDom: false,
```

Comment:
Does it need deprecating if it was only an experimental setting in the first place?

Reply:
This was never documented and only made available for a select few devs to help out with adding shadowdom support PR:s and setting it wouldn't break anything if we have shadowdom support out of the box so I suggest just remove all existence of it.

Reply:
Per discussion on Slack, we've decided to keep the deprecated flag.